app/dashboard/__init__.py

```diff
@@ -315,6 +318,9 @@ def ajax_compare(doc_id, api):
     "/_ajax/<string:resource>/distinct/<string:field>/", methods=["GET"])
 def ajax_distinct(resource, field):
     if validate_csrf(request.headers.get(CSRF_TOKEN_H, None)):
+        # Workaround because the backend uses /test/suite/...
+        if (resource == "suite"):
+            resource = "test/suite"
```

Comment:
Yes I did, it's not caught because the regex uses "/" as delimiters, sending one in the ajax call breaks the regex link and is not caught: E404. 
I could add an optional "/" in the regex for resource and it might work, I you prefer. It's a another solution.

Reply:
Nah... that's fine, I was more wondering if it magically worked correctly. ;-)